turbopack/crates/turbo-tasks-fs/src/retry.rs

```diff
@@ -23,21 +22,14 @@ where
     }
 }
 
-pub(crate) async fn retry_blocking<R, F>(
-    path: impl AsRef<Path>,
-    span: tracing::Span,
-    func: F,
-) -> io::Result<R>
+pub(crate) async fn retry_blocking<R, F>(path: impl AsRef<Path>, func: F) -> io::Result<R>
 where
     F: Fn(&Path) -> io::Result<R> + Send + 'static,
     R: Send + 'static,
 {
     let path = path.as_ref().to_owned();
 
-    let current_span = tracing::Span::current();
-    asyncify(move || {
-        let _entered = current_span.entered();
-        let _entered = span.entered();
```

Comment:
I think this logic with `current_span` was to ensure that the parent span is inherited from the thread spawning the work. Without this it looks like the span created in the callsites won't have a parent? (looking specifically at `"read directory"` and `"write symlink"`?)

Reply:
I used `turbo_tasks::spawn_blocking` which takes care of inheriting the current Span into the spawn_blocking context. Any spans created within `func` will have the correct parent span.